docs/application/dotnet/index.md

```diff
@@ -64,8 +64,8 @@ Tizen .NET consists of the following main components:
 
 Visual Studio Tools for Tizen provides Tizen-specific tools to improve your productivity.
 
-You can create a Tizen .NET application project with the Project Wizard in Visual Studio Tools for Tizen. When you create a new project with a specific template, the Project Wizard uses it to automatically create basic functionalities and default project files and folders for the application.
+You can create a Tizen .NET application project with the Project Wizard tool in Visual Studio Tools for Tizen. When you create a new project with a specific template, Project Wizard uses it to automatically create basic functionalities, and default project files and folders for the application.
 
-[![Download](media/ic_docs_download.png)](https://marketplace.visualstudio.com/items?itemName=tizen.VSToolsforTizen){:target="_blank"} The Visual Studio Tools for Tizen extension is registered in the Visual Studio Marketplace. You can [install the extension](../vstools/install.md) from the Visual Studio Marketplace in the Visual Studio IDE. To download the latest Visual Studio Tools for Tizen click the button. To use Visual Studio 2017, download [Visual Studio Tools for Tizen for Visual Studio 2017](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen).
+The Visual Studio Tools for Tizen extension is registered in Visual Studio Marketplace. You can [install the extension](../vstools/install.md) from Visual Studio Marketplace in the Visual Studio IDE. To download Visual Studio Tools for Tizen click [![Download](media/ic_docs_download.png)](https://marketplace.visualstudio.com/items?itemName=tizen.VSToolsforTizen). To use the latest version Visual Studio, download [Visual Studio Tools for Tizen for Visual Studio](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen).
 
```

Comment:
Fixed. Please verify.

Reply:
Thanks for applying the comment.

\[Visual Studio Tools for Tizen for Visual Studio]

I have an opinion for the later "for Visual Studio".

As the original sentence was to emphasize version (for Visual Studio 2017),
how about changing the "for Visual Studio"? For now "for Visual Studio" is redundant.
For example

\[Visual Studio Tools for Tizen for latest Visual Studio]
or
\[Visual Studio Tools for Tizen for Visual Studio 2019]

@hakkyu-samsung Would you please let me know your opinion?

Reply:
Hello, how about switching the content to:

The Visual Studio Tools for Tizen extension is registered in the Visual Studio Marketplace. 
You can learn how to install Visual Studio Tools for Tizen in [installing Visual Studio Tools for Tizen](../vstools/install.md) for both Visual Studio 2017 and 2019.

I think these sentences are enough to explain the key information from the original paragraph.
@surbhig12 and @mijin85cho, please let me know your thoughts and comments on this.

Reply:
@hakkyu-samsung,

In my opinion it would be good to provide the links. We can modify as suggested by @mijin85cho. Please let me know your views:

The Visual Studio Tools for Tizen extension is registered in Visual Studio Marketplace. You can [install the extension](../vstools/install.md) from Visual Studio Marketplace in the Visual Studio IDE. To download Visual Studio Tools for Tizen for Visual Studio 2017 click [![Download](media/ic_docs_download.png)](https://marketplace.visualstudio.com/items?itemName=tizen.VSToolsforTizen). To use the latest version Visual Studio, download [Visual Studio Tools for Tizen for Visual Studio 2019](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen).

Reply:
Perhaps the sentence should be "To use the latest version of Visual Studio, download Visual Studio Tools for Tizen for Visual Studio 2019"?

Everything else looks good.

Reply:
@hakkyu-samsung, yes, thank you for the correction:


The Visual Studio Tools for Tizen extension is registered in Visual Studio Marketplace. You can [install the extension](../vstools/install.md) from Visual Studio Marketplace in the Visual Studio IDE. To download Visual Studio Tools for Tizen for Visual Studio 2017 click [![Download](media/ic_docs_download.png)](https://marketplace.visualstudio.com/items?itemName=tizen.VSToolsforTizen). To use the latest version of Visual Studio, download [Visual Studio Tools for Tizen for Visual Studio 2019](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen).